plugin.xml

```diff
@@ -18,7 +18,7 @@
 
 
     <platform name="android">
-        <dependency id="cordova-plugin-fingerprint-aio" url="https://github.com/OutSystems/cordova-plugin-fingerprint-aio.git#4.0.1-OS6" />
+        <dependency id="cordova-plugin-fingerprint-aio" url="https://github.com/OutSystems/cordova-plugin-fingerprint-aio.git#fix/RMET-2652/biometrics-error-messages" />
```

Comment:
We can probably wait until we have a new version of `cordova-plugin-fingerprint-aio`, update it here, and merge the PR with that.

Reply:
Understood! Can merge this one here https://github.com/OutSystems/cordova-plugin-fingerprint-aio since it has a few approvals

Reply:
The other one was merged so I think we can cut a new release on it and update it here.